index.html

```diff
@@ -2,19 +2,199 @@
+                           </div>
+                        </div>
+                     </div>
+                  </div>
+</div>
 
-    <!-- Add your markup here -->
 
-</body>
 
+                     <div class="row">
+                        <div class="col-2">
+                           <div class="service-btn-black">
+                              <button class="button">Read More</button>
+                           </div>
+                        </div>
+                     </div>
+                  </div>
+               </div>
+            </div>
+         </div>
+      </div>
+   </section>
+   <!-- SECTION-5 // FOOTER WITH SOLID-COLOR-BACKGROUND WITH 2 COLUMN - LEFT COLUMN WITH CONTACT DETAILS+ADDRESS - RIGHT COLUMN - FORM -->
+   <!-- 
+      <footer>
+         <div class="row col-12 sm-col-12 md-col-12 lg-col-12 nav-bar-top">
+            <div class="row col-6 sm-col-6 md-col-6 lg-col-6 footer-address">
+               <div class="contact-me">
+                  <div class="hr">
+                     <hr class="small-2">
+                  </div>
+                  <h2>CONTACT</h2>
+               </div>
+               <div class="address-para">
+                  <p>500 Francisco, CA 98765</p>
+                  <p>Tel: 123-456-789</p>
+               </div>
+               <div class="address-para">
+                  <p>Fax: 123-456-789</p>
+                  <p>info@mysite.com</p>
+               </div>
+               <div class="social-media-links-row">
+                  <ul>
+                     <li class="facebook">
+                        <a href="#" class="fa fa-facebook"></a>
+                     </li>
+                     <li class="twitter">
+                        <a href="#" class="fa fa-twitter"></a>
+                     </li>
+                     <li class="instagram">
+                        <a href="#" class="fa fa-instagram"></a>
+                     </li>
+                     <li class="youtube">
+                        <a href="#" class="fa fa-youtube"></a>
+                     </li>
+                  </ul>
+               </div>
+               <div class="copy">
+                  <p>&copy; 2015 by Personal Life Coach. <br> Proudly created by Sonjide Hussain</p>
+               </div>
+            </div>
+            <div class="row col-6 sm-col-6 md-col-6 lg-col-6 footer-form">
+               <form class="contact-form-content">
+                  <label for="name">Enter Your Name*</label><br/>
+                  <input type="text" id="name" placeholder="   Name" /><br/>
+                  <label for="email">Enter Your Email*</label><br />
+                  <input type="text" id="email" placeholder="   Email" /><br/>
+                  <label for="phone">Enter Your Subject*</label><br />
+                  <input type="text" id="subject" placeholder="   Subject" /><br/>
+                  <label for="message">Message</label><br/>
+                  <input
```

Comment:
If you want a multi-line input box, you can use `<textarea>` instead of `<input>`, e.g.:

```html
<textarea id="message" placeholder="Message"></textarea>
```